src/Php81/README.md

```diff
@@ -4,6 +4,7 @@ Symfony Polyfill / Php80
 This component provides features added to PHP 8.1 core:
 
 - [`array_is_list`](https://php.net/array_is_list)
+- [`MYSQLI_REFRESH_REPLICA`](https://www.php.net/manual/en/mysqli.constants.php#constantmysqli-refresh-replica) constant
```

Comment:
This anchor does not exist in documentation yet but should be there soon after PHP 8.1 is released.